### PR TITLE
Implements (full|zeros|ones)

### PR DESCRIPTION
**Pull request by kaushikcfd**

### CODE REVIEW

File: pytato/array.py [hunk 1]
```diff
@@ -1738,4 +1738,34 @@ def log10(x: Array) -> IndexLambda:
...
+def zeros(namespace: Namespace, shape: ConvertibleToShape, dtype: type = float,
+        order: str = "C") -> Array:
+    # https://github.com/python/mypy/issues/3186
+    return full(namespace, shape, 0, dtype)  # type: ignore
+
+
+def ones(namespace: Namespace, shape: ConvertibleToShape, dtype: type = float,
+        order: str = "C") -> Array:
+    # https://github.com/python/mypy/issues/3186
+    return full(namespace, shape, 1, dtype)  # type: ignore
```
Comment (inducer) [inline pytato/array.py:1767]: Add to docs?

**Review by inducer — APPROVED** [review]
Thanks! LGTM other than docs.

File: pytato/array.py (hunk 1 above)
Comment (kaushikcfd) [reply pytato/array.py:1789]: `:'<,'>s/"\(\w\+\)", /.. autofunction:: \1\r/g`: 5e806aa

File: pytato/array.py [hunk 2]
```diff
@@ -1738,4 +1754,40 @@ def log10(x: Array) -> IndexLambda:
...
+    if order != "C":
+        raise ValueError("Only C-ordered arrays supported for now.")
+
+    shape = normalize_shape(shape, namespace)
+    fill_value = dtype(fill_value)
+    dtype = np.dtype(dtype)
+    return IndexLambda(namespace, fill_value, shape, dtype, {})
+
+
+def zeros(namespace: Namespace, shape: ConvertibleToShape, dtype: type = float,
```
[the remark this answers is not in the document]
Comment (inducer) [reply pytato/array.py]: That's fine.

**Comment by inducer** [comment]
Thanks!

**Pull request merged**